src/EnergyPlus/CurveManager.cc

```diff
@@ -2494,6 +2494,11 @@ namespace Curve {
 
                 if (normalizeMethod != NM_NONE && fields.count("normalization_divisor")) {
                     normalizationDivisor = fields.at("normalization_divisor").get<Real64>();
+                    if (fabs(normalizationDivisor) < std::numeric_limits<Real64>::min()) {
```

Comment:
std::abs

Reply:
In c++ you have function overloading, so std::abs is sufficient.

In C, abs takes an int, fabs a double, which is a common cause of problems.